packages/next-server/package.json

```diff
@@ -25,7 +25,6 @@
     ]
   },
   "dependencies": {
-    "@babel/runtime-corejs2": "^7.1.2",
```

Comment:
`@babel/runtime-corejs2` needs to be a production dependency, according to this : https://babeljs.io/docs/en/babel-plugin-transform-runtime#installation

Maybe there is a more specific use case here ? But it seems that `@babel/runtime-corejs2` will be required during runtime so if it's missing the app won't work